app/helpers/weams_helper.rb

```diff
@@ -0,0 +1,2 @@
+module WeamsHelper
```

Comment:
Nothing, just an artifact of rails g resource before I changed the app around. Will remove ...